css/components/alert.css

```diff
@@ -0,0 +1,84 @@
+@import "../base/all.css";
+
+.dflex{
+    display: flex;
+    align-items: center;
+}
+
+.flex-y{
+    flex-direction: column;
+}
+
+.flex-ai-start{
+    align-items: flex-start;
+}
+
+.m-left-auto{
+    margin-left: auto;
+}
+
+.alert{
+    padding:16px;
+    margin:16px;
+    border:1px solid transparent;
+    word-wrap: break-word;
+}
+
+.alert-info-icon{
+    margin-right: 16px;
```

Comment:
Suggestion: Try to use all the units in rem instead of px.

Reply:
Sure, I will use rem instead of px.

Reply:
Fixed: replaced px with rem.